warehouse/models/payments_views/payments_rides.sql

```diff
@@ -18,7 +19,16 @@
     filter_column = 'participant_id',
     filter_value = 'clean-air-express',
     principals = ['serviceAccount:clean-air-payments-user@cal-itp-data-infra.iam.gserviceaccount.com']
-) }}"
+) }}",
+" {{ create_row_access_policy(
+    principals = ['serviceAccount:metabase@cal-itp-data-infra.iam.gserviceaccount.com',
+                  'group:cal-itp@jarv.us',
+                  'domain:calitp.org',
+                  'user:angela@compiler.la',
+                  'user:easall@gmail.com',
+                  'user:jeremyscottowades@gmail.com',
```

Comment:
You mentioned possibly creating a group for payments. Is it possible to do that at least for these non-jarvus people? As time goes on and people shift off the project, I feel like things like this can become a security headache.

Reply:
Unfortunately I can't; right now, BigQuery only supports users (and service accounts), groups i.e. a group of emails in the same organization, and domains. I would prefer if we could use an IAM role but that's not supported (yet).